### DISCUSSION TITLE
타입스크립트 지원 여부

### DISCUSSION THREAD

**Discussion by zerone-jang** [zeakd/react-naver-maps#103] [General]
안녕하세요.
네이버 맵의 리액트 버전 잘 쓰고 있습니다.
혹시 타입스크립트 지원 계획이 있을까요?
타입스크립트 프로젝트에서는 사용할 수 없어서 아쉽습니다.

**Comment by zeakd** [comment]
안녕하세요. 혹시 몇버전을 사용중이실까요? 현재 최신 0.1.2 버전은 타입스크립트를 지원하고있습니다.

**Reply by zerone-jang** [reply]
아 제가 이전 버전 사용하고 있었네요.
다른 질문으로, 기본 naver map API의 모든 기능들이 구현되어 있는 상태인지 궁금합니다!

**Reply by zeakd** [reply]
Overlay와 Map만 구현되어있습니다. 나머지 요소들은 추가예정입니다만, useNavermaps등을 통해 직접 구현또한 가능합니다
https://zeakd.github.io/react-naver-maps/guides/customize-overlays/

**Reply by zerone-jang** [reply]
감사합니다!
